python/tvm/relay/frontend/tflite.py

```diff
@@ -2831,17 +2831,94 @@ def convert_transpose_conv(self, op):
         else:
             padding = (0, 0, 0, 0)
 
-        out = _op.nn.conv2d_transpose(
-            in_expr,
-            weight_expr_iohw,
-            strides=(stride_h, stride_w),
-            padding=padding,
-            channels=int(out_channels),
-            kernel_size=(int(kernel_h), int(kernel_w)),
-            data_layout="NHWC",
-            kernel_layout="OIHW",
-            out_dtype=output_tensor_type_str,
-        )
+        if input_tensor.qnn_params:
+            # Making use of qnn.conv2d
```

Comment:
May be useful to document the mathematical approach taken here.

Reply:
This is essentially the same as the implementation of Relay op `conv2d_transpose`, which makes the same transformations (with 0 instead of zero-point) to the `input `and `kernel `and then does a convolution. So, the mathematical approach would rather be proving that the transformations followed by convolution is the same as transpose convolution, proving which might take some unreasonable space.

Reply:
I suppose I was less interested in a proof, and more just a statement of what manipulations are happening.

Reply:
Okay, added that.